config/jobs/kubernetes/sig-testing/kubetest2-canaries.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes/test-infra:
+  - name: kubetest2-gce-deployer-build-up-down
```

Comment:
maybe a simpler name `kubetest2-gce-deployer-smoke` ?

Reply:
Hmm. I think I favor the current name because I can see a world in which there are more presubmits for kt2 and "smoke" is very generic. Fine either way.

Reply:
this would be gce deployer specific though right? also consider dropping "deployer" 
`kubetest2-gce-(smoke|e2e|buildupdown)` ? either is fine.

Reply:
Dropped deployer and I'll leave it with `build-up-down`. Should be good to go.